### PR TITLE
Fix/as 115 backlink without js

### PR DESCRIPTION
**Pull request by a6software**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-115

## Description of change
Back link should work when Javascript is disabled

PR is done, but a bug has been spotted during testing with an indirect dependency. AS-1854

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/views/layouts/main.njk [hunk 1]
```diff
@@ -41,7 +41,7 @@
       {% block backButton %}
         {{ govukBackLink({
           text: "Back",
-          href: "javascript:history.back()",
+          href: navigation[1],
```
Comment (a6software) [inline packages/forms-web-app/src/views/layouts/main.njk:44]: the middleware ensures there will always be at least two items in the navigation history - the current page, and the fallback page.

There is an edge case where this would break, and that would be if we showed the back button on the fall back page. This should never happen, though if it ever does, it should be caught by this test:

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/919/files#diff-2f8b09958305a61fd225779a562fa8dcd065b8572ae31fe6f072e36282c4fcf1R74

File: e2e-tests/cypress/integration/eligibility-listed-building-status.feature [hunk 2]
```diff
@@ -2,15 +2,16 @@
 Feature: A prospective appellant states whether or not their appeal covers a listed building
     Our service doesn't cover listed buildings.
 
-  @as-1670
-  Scenario: Listed Building statement is required
-    When the user does not provide a Listed Building statement
-    Then the user is informed that a Listed Building statement is required
+#  @as-1670
```
Comment (a6software) [inline e2e-tests/cypress/integration/eligibility-listed-building-status.feature:5]: this should not be commented out

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Comment by a6software** [comment]
ty
